vm/src/obj/objint.rs

```diff
@@ -15,6 +15,22 @@ use std::hash::{Hash, Hasher};
 // This proxy allows for easy switching between types.
 type IntType = BigInt;
 
+pub type PyInt = BigInt;
+
+impl IntoPyObject for PyInt {
+    fn into_pyobject(self, ctx: &PyContext) -> PyResult {
+        Ok(ctx.new_int(self))
+    }
+}
+
+// TODO: macro to impl for all primitive ints
```

Comment:
You could also do `impl IntoPyObject for Into<BigInt>` right? Then the trait system would solve this duplication.

Reply:
Yeah, I tried this initially but sadly it doesn't work in this case because it conflicts with the `bool` impl.

```
error[E0119]: conflicting implementations of trait `pyobject::IntoPyObject` for type `bool`:
  --> vm/src/obj/objint.rs:26:1
   |
26 | impl<T: Into<BigInt>> IntoPyObject for T {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `bool`
   | 
  ::: vm/src/obj/objbool.rs:8:1
   |
8  | impl IntoPyObject for bool {
   | -------------------------- first implementation here
```